src/Detector/Detector.csproj

```diff
@@ -35,18 +35,26 @@
     <PackageProjectUrl>https://github.com/microsoft/oryx</PackageProjectUrl>
     <PackageTags>Oryx Detector OryxDetector</PackageTags>
     <Description>A library to detect tools and frameworks used in an application.</Description>
+    <Version>1.0.0-pre-</Version>
```

Comment:
can we just have latest here?

Reply:
Prerelease packages on NuGet.org typically start with `pre`, so I am using that convention here.